test/utils/json.test.js

```diff
@@ -0,0 +1,84 @@
+const { prettyPrint } = require('../../src/utils/json');
+
+describe('utils > json', () => {
+  it('prettyPrint simple primitive types', () => {
```

Comment:
Maybe we could continue using `should` to display `it should ...`. Not sure though. You could also add another describe to cover `prettyPrint` part only.

```suggestion
  it('should pretty print simple primitive types', () => {
```